### ISSUE TITLE
import data and configuration issue 

### ISSUE THREAD

**Issue by shreo1992-beep** [UniTime/unitime#124]
Dears

kindly i want to ask about some features that can not be imported in xml or csv using Data Exchange so if i want to import for example Room type data weather in xml or csv can this be customizable or not and if it is not is it possible to import it to unitime without creating it manually , So if i have xml file or csv file like this 

[RoomType.xml.xlsx](https://github.com/UniTime/unitime/files/9468994/RoomType.xml.xlsx)
thanks & best regards

**Comment by tomas-muller** [comment]
The available XML imports are listed at [www.unitime.org/uct_interfaces.php](https://www.unitime.org/uct_interfaces.php). We do not have an import for lookup tables like room types, room feature types, etc. as these are usually only needed to be configured once and only contain a small number of records.

That said, it is possible to extend or customize UniTime to add such imports. There is currently no plan in doing so, however, so any contribution in this regard would be appreciated.

**Comment by shreo1992-beep** [comment]
@tomas-muller is it possible to do it as a script

**Comment by tomas-muller** [comment]
Yes, it can be done as a script using the [Scripts](https://help.unitime.org/scripts) page.
